public/app/features/admin/state/actions.ts

```diff
@@ -35,7 +35,7 @@ export function loadAdminUserPage(userId: number): ThunkResult<void> {
       await dispatch(loadUserProfile(userId));
       await dispatch(loadUserOrgs(userId));
       await dispatch(loadUserSessions(userId));
-      if (config.ldapEnabled && config.licenseInfo.hasLicense) {
+      if (config.ldapEnabled && featureEnabled('ldapsync')) {
```

Comment:
Nit. In general I think that maybe the backend should provide to the frontend if ldapsync is enabled based on licensing and whether ldap is acually enabled to get rid of checking multiple flags in frontend. But probably a bigger change required.

Reply:
it's a good comment, and I had the same thought regarding the relationship between features being enabled via config, feature toggles and/or licensing.  This PR has already grown quite large so I figure we can figure out whether it makes sense to combine them as a separate issue.

Reply:
Great. Make sense